### PR TITLE
Added limits to admin panel

### PR DESCRIPTION
**Pull request by Jozefik**
Fixed issue #1077

### CODE REVIEW

File: forge/routes/api/admin.js [hunk 1]
```diff
@@ -4,13 +4,18 @@ module.exports = async function (app) {
     app.get('/stats', async (request, reply) => {
         const userCount = await app.db.models.User.count({ attributes: ['admin'], group: 'admin' })
         const projectStateCounts = await app.db.models.Project.count({ attributes: ['state'], group: 'state' })
+        const license = await app.license.get() || {}
```
Comment (hardillb) [inline forge/routes/api/admin.js:7]: I feel we should find a way to populate this with the default values if no license present, then we don't have hard coded values in the UI portion that we may need to change later.

**Comment by hardillb** [comment]
Sorry, meant to say thanks before suggesting changes....

File: forge/routes/api/admin.js (hunk 1 above)
Comment (hardillb) [reply forge/routes/api/admin.js:7]:
```suggestion
        const license = await app.license.get() || app.license.defaults
```

Comment (Jozefik) [reply forge/routes/api/admin.js]: Sure i will change it. Thanks for the suggestion.

**Comment by hardillb** [comment]
@knolleary happy for me to merge this? All tests have passed.

**Review by knolleary — APPROVED** [review]

**Pull request merged**